community-templates.json

```diff
@@ -0,0 +1,19 @@
+[
+  {
+    "name": "a-sample-test-template",
```

Comment:
We need some community template example. We do not have any so I think that having JS file will be better:

- we can put comment with example (as documentation)
- We can leave that empty for the moment

Reply:
Yeah I'll leave this file empty for now. I only added this as a refernence. When I write the docs I'll mention the way and syntax of adding templates.

Reply:
yes!